### PR TITLE
Update nodejs InfoResponse

### LINKED ISSUE #1016: Update InfoResponse - nodejs
https://github.com/iotaledger/tips-draft/pull/24

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update InfoResponse according to https://github.com/iotaledger/tips-draft/pull/24/files

## Links to any relevant issues

Closes #1016

### CODE REVIEW

File: bindings/nodejs/lib/types/models/info/node-info-base-token.ts [hunk 1]
```diff
@@ -17,14 +17,14 @@ export interface INodeInfoBaseToken {
...
-     */
-    subunit?: string;
     /**
      * The base token decimals.
      */
     decimals: number;
+    /**
+     * The base token sub-unit.
+     */
+    subunit?: string;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-base-token.ts:27]: 😠

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/models/info/node-info-base-token.ts:27]: I'd rather change that in the TIP...

Comment (Thoralf-M) [reply bindings/nodejs/lib/types/models/info/node-info-base-token.ts:27]: Then do it :trollface:

File: bindings/nodejs/lib/types/models/info/node-info-status.ts [hunk 2]
```diff
@@ -10,7 +10,39 @@ export interface INodeInfoStatus {
      */
     isHealthy: boolean;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-status.ts:11]: Maybe also take the doc from the TIP for this one

File: bindings/nodejs/lib/types/models/info/node-info-status.ts [hunk 3]
```diff
@@ -10,7 +10,39 @@ export interface INodeInfoStatus {
...
+     */
+    relativeConfirmedTangleTime?: string;
+    /**
+     * The latest slot that the node has committed to.
+     */
+    latestCommitmentId: string;
+    /**
+     * The index of latest finalized slot.
+     */
+    latestFinalizedSlot: string;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-status.ts:35]: Any way to use proper types like SlotIndex here or is that later converted to?

File: bindings/nodejs/lib/types/models/info/node-info-base-token.ts (hunk 1 above)
Comment (thibault-martinez) [reply bindings/nodejs/lib/types/models/info/node-info-base-token.ts:27]: Asked, will resolve this for now

File: bindings/nodejs/lib/types/models/info/node-info-status.ts (hunk 3 above)
Comment (Thoralf-M) [reply bindings/nodejs/lib/types/models/info/node-info-status.ts:35]: SlotIndex is a bigint, not sure if that can be used if it's a string

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Failing CI

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 4]
```diff
@@ -1,29 +1,222 @@
...
+     */
+    manaDecayFactorEpochsSum: number;
+    /**
+     * The scaling of ManaDecayFactorEpochsSum expressed as an exponent of 2.
+     */
+    manaDecayFactorEpochsSumExponent: number;
+    /**
+     * The unbonding period in epochs before an account can stop staking.
+     */
+    stakingUnbondingPeriod: number;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: string?

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 5]
```diff
@@ -1,29 +1,222 @@
...
+     */
+    decreaseThreshold: number;
+    /**
+     * SchedulerRate is the rate at which the scheduler runs in workscore units per second.
+     */
+    schedulerRate: number;
+    /**
+     * MinMana is the minimum amount of Mana that an account must have to have a block scheduled.
+     */
+    minMana: number;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: string?

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 6]
```diff
@@ -1,29 +1,222 @@
...
+ */
+export interface VersionSignalingParameters {
+    /**
+     * The size of the window in epochs to find which version of protocol parameters was most signaled, from currentEpoch - windowSize to currentEpoch.
+     */
+    windowSize: number;
+    /**
+     * The target number of supporters for a version to win in a windowSize.
+     */
+    windowTargetRatio: string;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: number?

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 7]
```diff
@@ -1,29 +1,222 @@
...
     /**
-     * The rent structure used by given node/network.
+     * The offset in epochs to activate the new version of protocol parameters.
      */
-    rentStructure: IRent;
+    activationOffset: string;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: number?

**Review by thibault-martinez — DISMISSED** [review]

File: bindings/nodejs/lib/types/models/info/node-info-metrics.ts [hunk 8]
```diff
@@ -6,15 +6,15 @@
...
+    blocksPerSecond: string;
     /**
-     * Referenced blocks per second.
+     * The current rate of confirmed blocks per second.
      */
-    referencedBlocksPerSecond: number;
+    confirmedBlocksPerSecond: string;
     /**
-     * The rate at which rates are being referenced.
+     * The ratio of confirmed blocks in relation to new blocks of the last confirmed slot.
```
Comment (Alex6323) [inline bindings/nodejs/lib/types/models/info/node-info-metrics.ts]:
```suggestion
     * The ratio of confirmed blocks to new blocks of the last confirmed slot.
```

Comment (Alex6323) [reply bindings/nodejs/lib/types/models/info/node-info-metrics.ts]: but anything is better than the previous doc :smile: (so ignore the suggestion if you want)

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/models/info/node-info-metrics.ts]: Not sure what it's worth but of/to seems preferred https://ell.stackexchange.com/questions/4279/when-describing-a-ratio-should-between-or-of-be-used

**Review by DaughterOfMars — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Pull request merged**